### DISCUSSION TITLE
Noice notif everytime I type in cmdline

### DISCUSSION THREAD

**Discussion by BSIT-Evanism** [LazyVim/LazyVim#633] [Q&A] [answered]
Hi, i just wanna ask the root of the error of this notif that spams everytime I type on the commandline, thank you and sorry for hassle
![image](https://user-images.githubusercontent.com/71175442/233144459-de8323c7-c7c5-4c7d-8c88-a9105221235d.png)

**Comment by amaanq** [comment, marked as answer]
update nvim, nvim-ts, and your parsers.
